### PR TITLE
Added link to https://learnrelay.org/ to README

### PR DESCRIPTION
**Pull request by dminkovsky**

### CODE REVIEW

**Comment by schickling** [comment]
Any update on merging this? 🎉 

Thanks for the PR @dminkovsky!

File: README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ Relay is a JavaScript framework for building data-driven React applications.
 * **Colocation:** Queries live next to the views that rely on them, so you can easily reason about your app. Relay aggregates queries into efficient network requests to fetch only what you need.
 * **Mutations:** Relay lets you mutate data on the client and server using GraphQL mutations, and offers automatic data consistency, optimistic updates, and error handling.
 
-[Learn how to use Relay in your own project.](https://facebook.github.io/relay/docs/getting-started.html)
+[Learn how to use Relay in your own project](https://facebook.github.io/relay/docs/getting-started.html) or read a [comprehensive introduction](https://learnrelay.org).
```
Comment (josephsavona) [inline README.md]: Let's add a note clarifying that Learn Relay is a community-driven project and not an official source of documentation. How about:

`[Learn how to use Relay in your own project](https://facebook.github.io/relay/docs/getting-started.html) or check out the community-driven introduction at [Learn Relay](https://learnrelay.org).`

Comment (schickling) [inline README.md]: Sounds good! Does @dminkovsky need to do the change or can you do it directly?

Comment (josephsavona) [inline README.md]: Would be easiest if @dminkovsky can update the PR directly.

**Comment by dminkovsky** [comment]
Maybe Johannes can update my PR? I don't have time right now unfortunately.
Thank you.

**Comment by schickling** [comment]
Created a new PR #1498 instead.

**Comment by josephsavona** [comment]
Thanks!

**Pull request closed without merging**